src/sidebar/templates/group-list.html

```diff
@@ -82,4 +84,132 @@
       </div>
     </li>
   </ul>
+
+  <!-- Show new menu if community_groups feature flag is enabled. -->
+  <div class="dropdown-menu" role="menu" ng-if="vm.showGroupsMenu() && vm.isFeatureFlagEnabled('community_groups')">
+
+    <!-- Display Currently Viewing. -->
+    <h2 class="dropdown-menu__section-heading" ng-show="vm.currentlyViewingGroupOrganizations().length > 0">Currently Viewing</h2>
+    <ul class="dropdown-menu__section" ng-show="vm.currentlyViewingGroupOrganizations().length > 0">
```

Comment:
Are each of these sections the same except for the input list of groups?

Reply:
I had a simillar thought. Yes except for My Groups which handles out of scope groups differently than the Currently Viewing section (There shouldn't be any out of scope groups in the Featured section.) I wasn't sure how to apply a loop over these neatly with that caveat. I suppose I could keep track of the actual list name and use an ng-if in there. Perhaps that's not _too_ complicated.

Reply:
I think it will be easier to do (and review) if the sub-list is broken out into a separate component, which can then be parametrized like a function. I would suggest to do that in a separate PR after this is merged though.